src/digisos/redux/soknad/soknadReducer.ts

```diff
@@ -169,6 +170,26 @@ export default (state: SoknadState = defaultState, action: SoknadActionType) =>
             };
         }
 
+		case SoknadActionTypeKeys.OPPDATER_SAMTYKKE: {
```

Comment:
Trenger vi å håndere denne når vi kun returnerer staten? Samme gjelder for `HENT_SAMTYKKE`

Reply:
Jeg tipper at du har rett.

Reply:
Jeg prøver å ta de bort (+ en til som jeg fant.)